synfig-studio/src/gui/canvasview.cpp

```diff
@@ -871,7 +846,7 @@ CanvasView::create_time_bar()
 	//Setup the Animation Mode Button and the Keyframe Lock button
 	{
 		Gtk::IconSize iconsize=Gtk::IconSize::from_name("synfig-small_icon_16x16");
-		Gtk::Image *icon = manage(new Gtk::Image(Gtk::StockID("synfig-animate_mode_off"), iconsize));
+		Gtk::Image *icon = manage(create_image_from_icon("animate_mode_off_icon", iconsize));
 		animatebutton = Gtk::manage(new class Gtk::ToggleButton());
```

Comment:
I suggest not creating an image widget, but just using the `animatebutton->set_image_from_icon_name(...)` method.

Reply:
Done